packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditReducers.ts

```diff
@@ -105,7 +105,8 @@ function generateLeafNodeSchemas(nodeTypes: string[]) {
 		if (
 			nodeType !== "treefuzz.node" &&
 			nodeType !== "com.fluidframework.leaf.number" &&
-			nodeType !== "com.fluidframework.leaf.string"
+			nodeType !== "com.fluidframework.leaf.string" &&
+			nodeType !== "com.fluidframework.leaf.handle"
 		) {
 			if (!nodeType.startsWith("com.fluidframework.leaf")) {
```

Comment:
This code is strange. Why is it testing for the node type not being one of these types, then testing for it not starting with "com.fluidframework.leaf"? THis first check for some (but not al) leaf types has no effect since they are filtered out by the second if.

This whole function is rather confusing: it takes a (mutable) array of type names. THen it assumes those names are fully qualified tree node type identifiers, compoares it to some other node identifiers, and if its not one of tose it guets used as a non-fully qualified iidentifer, getting prefixed with "com.fluidframework.leaf"? THis function needs more docs that just a TODO as its really unclear whats going on here.

Anyway, the change made to this in this PR has no impact since "com.fluidframework.leaf.handle" starts with "com.fluidframework.leaf". TRhe line before it also does nothing. I'm not sure what aspect of this function is the bug since I don't know what its supposed to do.

Reply:
@jzaffiro is this change needed? @CraigMacomber  seems to imply that we could just not change this, but i'm not sure. regardless i think we should just file a bug, and do the minimal change to get this working

Reply:
Removing the line I added has no effect, so I'll remove it. Created [AB#7879](https://dev.azure.com/fluidframework/internal/_workitems/edit/7879) to track the documentation and clean up of the function.